core/src/main/java/org/apache/cxf/attachment/AttachmentUtil.java

```diff
@@ -75,6 +75,8 @@
     private static final Random BOUND_RANDOM = new Random();
     private static final CommandMap DEFAULT_COMMAND_MAP = CommandMap.getDefaultCommandMap();
     private static final MailcapCommandMap COMMAND_MAP = new EnhancedMailcapCommandMap();
+    
+    private static final String BINARY = "binary";
```

Comment:
make it package visible and reuse in AttachmentSerializer like BODY_ATTACHMENT_ID

Reply:
@amarkevich  . This is done with thanks